README.md

```diff
@@ -96,7 +96,65 @@ Add Stumbles
             ]
           },
           type : "POST",
-          success : function(r) {
-            console.log(r);
+          success : function(req) {
+            console.log(req);
+          }
+        });
+
+Get Leaders
+----
+  Get the leaders for a given Country.
+
+* **URL**
+
+  https://leaderboard.services.mozilla.com/backend/leaders
+
+* **Method:**
+
+  `GET`
+
+*  **URL Params**
+
+  * country_id : `int`
+
+* **Data Params**
+
+  None
+
+* **Request Headers**
+
+  None
+
+* **Success Response:**
+
+  * **Code:** 200
+
+  JSON encoding
+
+        {
+          country_name: <str>,
+          leaders: [
+            {
+              name: <str>,
+              observations: <int>
+            }
+          ]
+        }
+
+* **Error Responses:**
+
+  None
+
+* **Sample Call:**
+
+        $.ajax({
+          url: "https://leaderboard.services.mozilla.com/backend/leaders",
+          dataType: "json",
+          data: {
+            country_id: 1,
+          }
+          type : "GET",
+          success : function(r, data) {
```

Comment:
I still think jquery is the wrong syntax to demonstrate this stuff.  I'd really rather see a simple curl statement as it : 

* cuts out oddities like the `success` callback and the console.log call which are unrelated to HTTP
* makes the headers like content-type explicit